packages/plugins/graphql/strapi-server.js

```diff
@@ -1,15 +1,20 @@
 'use strict';
 
+const bootstrap = require('./server/bootstrap');
+const register = require('./server/register');
+const services = require('./server/services');
+
 module.exports = (/* strapi, config */) => {
   return {
-    bootstrap: () => {},
-    destroy: () => {},
-    config: {},
-    routes: [],
-    controllers: {},
-    services: () => {},
-    policies: {},
-    middlewares: {},
-    contentTypes: {},
+    bootstrap,
+    register,
+    services,
+    // destroy: () => {},
+    // config: {},
+    // routes: [],
+    // controllers: {},
+    // policies: {},
+    // middlewares: {},
+    // contentTypes: {},
```

Comment:
you can remove the comments if not needed

Reply:
It has been removed in another PR